src/utils/components/EvictionStrategy/EvictionStrategyModal.tsx

```diff
@@ -26,17 +30,30 @@ const EvictionStrategyModal: React.FC<EvictionStrategyModalProps> = ({
   vmi,
 }) => {
   const { t } = useKubevirtTranslation();
-  const [checked, setChecked] = React.useState<boolean>(
-    !!vm?.spec?.template?.spec?.evictionStrategy,
-  );
 
-  const updatedVirtualMachine = React.useMemo(() => {
+  const [checked, setChecked] = useState<boolean>(true);
```

Comment:
nitpick: We typically use the `is<Whatever>` convention for boolean state. I would make this `isChecked` to remain consistent. There are other places where we use `checked`, but I think we should refactor those to match the convention.